aspnetcore/release-notes/aspnetcore-5.0.md

```diff
@@ -0,0 +1,476 @@
+
+Catch-all route parameters, which capture paths across multiple folder boundaries, are supported in components. For more information, see <xref:blazor/fundamentals/routing#catch-all-route-parameters>.
+
+### Debugging improvements
+
+Debugging Blazor WebAssembly apps is improved in ASP.NET Core 5.0. Additiionally, debugging is now supported on Visual Studio for Mac. For more information, see <xref:blazor/debug>.
+
+### Microsoft Identity v2.0 and MSAL v2.0
+
+Blazor security now uses Microsoft Identity v2.0 ([`Microsoft.Identity.Web`](https://www.nuget.org/packages/Microsoft.Identity.Web) and [`Microsoft.Identity.Web.UI`](https://www.nuget.org/packages/Microsoft.Identity.Web.UI)) and MSAL v2.0. For more information, see the topics in the [Blazor Security and Identity node](xref:blazor/security/index).
+
+### State management support
+
+`Microsoft.AspNetCore.ProtectedBrowserStorage` has been moved to `Microsoft.AspNetCore.Components.Web.Extensions` and has undergone API changes. State management documentation has been split across Blazor hosting models. For more information, see <xref:blazor/state-management>.
+
+<!-- WAIT ON THIS ENTRY UNTIL https://github.com/dotnet/AspNetCore.Docs/pull/19887 MERGES.
+
+### Component integration across hosting models
+
+Component integration is improved across hosting models.
+
+For more information, see <xref:blazor/components/integrate-components-into-razor-pages-and-mvc-apps>.
+
+-->
+
+### Linker is now a Trimmer
+
+Prior to the release of ASP.NET Core 5.0, Blazor WebAssembly performs Intermediate Language (IL) linking during a build to trim unnecessary IL from the app's output assemblies. After the release of 5.0, Blazor WebAssembly performs *trimming* to reduce the size of the published output. For more information, see <xref:blazor/host-and-deploy/configure-trimmer>.
+
+### Browser compatibility analyzer
+
+Blazor WebAssembly apps target the full .NET API surface area, but not all .NET APIs are supported on WebAssembly due to browser sandbox constraints. Unsupported APIs throw <xref:System.PlatformNotSupportedException> when running on WebAssembly. A platform compatibility analyzer warns the developer when the app uses APIs that aren't supported by the app's target platforms. For more information, see <xref:blazor/components/class-libraries#browser-compatibility-analyzer-for-blazor-webassembly>.
+
+### Lazy load assemblies
+
+Blazor WebAssembly app startup performance can be improved by deferring the loading of some application assemblies until they are required. For more information, see <xref:blazor/webassembly-lazy-load-assemblies>.
+
+### Updated globalization support
+
+Globalization support is available for Blazor WebAssembly based on International Components for Unicode (ICU). For more information, see <xref:blazor/globalization-localization>.
+
+## gRPC
+
+Many preformance improvements have been made in [gRPC](https://grpc.io/). <!-- For more information, see [this ]
```

Comment:
gRPC perf blog post is now live: https://devblogs.microsoft.com/aspnet/grpc-performance-improvements-in-net-5/